app/inputs/attachment_preview_input.rb

```diff
@@ -0,0 +1,9 @@
+class AttachmentPreviewInput < SimpleForm::Inputs::FileInput
+  def input(wrapper_options = nil)
+    out = ''
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.